src/component/archive/ArchiveTable/ArchivedFeatureActionCell/ArchivedFeatureDeleteConfirm/ArchivedFeatureDeleteConfirm.tsx

```diff
@@ -0,0 +1,99 @@
+import { Alert, styled } from '@mui/material';
+import React, { useState } from 'react';
+import { Dialogue } from 'component/common/Dialogue/Dialogue';
+import Input from 'component/common/Input/Input';
+import { IFeatureToggle } from 'interfaces/featureToggle';
+import { formatUnknownError } from 'utils/formatUnknownError';
+import { useFeatureArchiveApi } from 'hooks/api/actions/useFeatureArchiveApi/useReviveFeatureApi';
+import useToast from 'hooks/useToast';
+
+interface ArchivedFeatureDeleteConfirmProps {
+    deletedFeature?: IFeatureToggle;
+    open: boolean;
+    setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    refetch: () => void;
+}
+
+const StyledDeleteParagraph = styled('p')(({ theme }) => ({
+    marginTop: theme.spacing(4),
+}));
+
+const StyledFormInput = styled(Input)(({ theme }) => ({
+    marginTop: theme.spacing(2),
+}));
+
+const ArchivedFeatureDeleteConfirm = ({
```

Comment:
The current approach we're trying to follow is to avoid default exports, so I suggest this change:
```suggestion
export const ArchivedFeatureDeleteConfirm = ({
```

And deleting the export default at the bottom of the file.

Reply:
Done